embedded-visualization/Virtual Machines/cspe-VM-Capacity-and-Cost.json

```diff
@@ -606,12 +605,12 @@
     ]
   },
   "time": {
-    "from": "now-30d",
+    "from": "now-6h",
```

Comment:
is this an intentional change?

Reply:
Yes and no... It doesn't matter, the default value should better stay 30d actually but it's something you can change live in the dashboard through the time picker at the top.

Reply:
so just revert it

Reply:
Understood!